src/RA_Dlg_Achievement.h

```diff
@@ -56,7 +56,7 @@ class Dlg_Achievements
     void RemoveAchievement(HWND hList, int nIter);
     size_t AddAchievement(HWND hList, const Achievement& Ach);
 
-    INT_PTR Dlg_Achievements::CommitAchievements(HWND hDlg);
+    INT_PTR CommitAchievements(HWND hDlg);
```

Comment:
`Error C4596 'CommitAchievements': illegal qualified name in member declaration`

It might have worked before because the compiler adjusted for it with an extention but putting the scoped name inside the same scope is wrong. It could have been mistaken as `Dlg_Achievements::Dlg_Achievements::CommitAchievements`.

Reply:
Most likely the result of copying the prototype from the cpp file and not stripping the class.